docs/admin/users/idp-sync.md

```diff
@@ -327,8 +327,7 @@ the OIDC provider. See
 > ones include `groups`, `memberOf`, and `roles`.
 
 Next configure the Coder server to read groups from the claim name with the
-[OIDC organization field](../../reference/cli/server.md#--oidc-organization-field)
-server flag:
+OIDC organization field server flag:
```

Comment:
This CLI arg got moved to runtime config a few days ago, so it's no longer in the reference. The docs will need updating soon.